domains/vulpine.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "VulpineFriend87",
+           "email": "alessandrobuday@gmail.com",
+           "discord": "911268706112397392"
+        },
+    
+        "record": {
+            "CNAME": "vulpinefriend87.github.io"
```

Comment:
Would like you to add more stuff to your website

Reply:
Oh because you can't update that?

Reply:
Can I ask for an update later?